src/signer/src/main/java/ee/ria/xroad/signer/tokenmanager/token/SoftwareTokenWorker.java

```diff
@@ -197,6 +195,7 @@ protected void deleteCert(String certId) throws Exception {
         Signature signature = Signature.getInstance(SIGNATURE_ALGORITHM);
```

Comment:
This is previous code. Signature mechanism of the software token is not configured via _device.ini_ configuration file. Software token uses always hardcoded _NONEwithRSA_ algorithm.